core-bundle/contao/templates/_new/content_element/gallery.html.twig

```diff
@@ -2,6 +2,8 @@
 {% use "@Contao/component/_list.html.twig" %}
 {% use "@Contao/component/_figure.html.twig" %}
 
+{% set attributes = attrs(attributes|default).addClass("cols-#{items_per_row|default(4)}") %}
```

Comment:
I‘m not sure about the class name because `cols-*` is also used by Bootstrap (and probably others). Should we pick a more unique name?

Reply:
`per-row-4` would match the `perRow` DB column name?
Or `content-gallery--cols-4` to be based on the class name of the element itself?

Reply:
I‘m fine with both, so unless one of the @contao/developers has a preference, you can choose either one.

Reply:
I’d go with `content-gallery--cols-4` then.

```suggestion
{% set attributes = attrs(attributes|default).addClass("content-#{type}--cols-#{items_per_row|default(4)}") %}
```

Reply:
I'm in favor of 💩, I mean BEM, although I have no strong opinion here (I am just used to using it now). However - may be we should go over all the classes in our current templates again to be mroe consistent with the usage of BEM? For example in the `text` template we just use the class `rte` for the wrapper containing the text. If we follow BEM it would be `content-text__rte`, etc.

Reply:
> For example in the `text` template we just use the class `rte` for the wrapper containing the text. If we follow BEM it would be `content-text__rte`, etc.

I’m not sure, the `rte` element can be used in other elements too so I would see it as it’s own block.

Reply:
I am assuming the `rte` class will also be used in the news templates at some point though. Currently they use `ce_text` for the teaser.